lib/determine-basal/determine-basal.js

```diff
@@ -374,10 +374,20 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
             }
         }
     }
+  
+    var minutes_running;
+    if (typeof currenttemp.duration == 'undefined' || currenttemp.duration == 0) {
+        minutes_running = 0;
```

Comment:
Thanks.  It matters because "temp not running" is represented (in OpenAPS) as rate 0, duration 0.  So the behavior you just fixed was causing it to calculate a non-zero lowtempimpact in that (common) situation.